index.html

```diff
@@ -36,14 +36,42 @@
 
 <p>This project is an implementation of the paper "Meshing piecewise linear complex using CDT" by Hang Si et. al. Constrained Delaunay tetrahedralization attempts to retain Delunay property for cells while ensuring preservation of constraint vertices, segments and facets. </p>
 
+<h1>
+<a id="terminology" class="anchor" href="#terminology" aria-hidden="true"><span class="octicon octicon-link"></span></a>Terminology</h1>
+
+<p>Delaunay: Let V be the set of vertices of X(a PLC). Let σ be any simplex formed by vertices of V . σ is delaunay if there exists a circumsphere of σ that encloses no vertex of V.</p>
+
```

Comment:
@HarshitJn I think we should define _strongly Delaunay_ as well which is a stricter version of Delaunay property, as you have referred it in definition of the term _Edge Protected_